docs/api/cameras/PerspectiveCamera.html

```diff
@@ -48,7 +48,11 @@
 
 
 		<h2>Properties</h2>
-		<div>See the base [page:Camera] class for common properties.</div>
+		<div>
+			See the base [page:Camera] class for common properties.<br>
+			Note that after making changes to most of these poperties you will have to call 
```

Comment:
>poperties

How about this, instead?

"Note that after making changes to a property that modifies the camera's frustum, you will have to call..."

Reply:
I added it like this since it's a likely source of confusion for people new to the library and this is the simplest wording possible. I'm open to changing it though if you prefer.

Reply:
As written, it is not clear to the reader exactly when to call `updateProjectionMatrix()` -- nor why.

IMHO, my suggestion is an improvement, but even better would be to list the properties you are referring to.

In any event, thank you for all your work on the docs. :)

Reply:
How about noting for each property that needs this in the property description?

Reply:
Yes, that would be best.